client/core/core.go

```diff
@@ -117,7 +117,8 @@ type dexConnection struct {
 
 	// tradeMtx is used to synchronize access to the trades map.
 	tradeMtx sync.RWMutex
-	trades   map[order.OrderID]*trackedTrade
+	// trades tracks outstanding orders issued by this client
```

Comment:
Srry for nitpick, but period at the end. I treat `trades` here as a proper noun although not capitalized, so grammatically correct to call it a sentence, with that one exception. I think.